third_party/softfloat/CMakeLists.txt

```diff
@@ -0,0 +1,413 @@
+####################################################################################################
+# Copyright 2023 Chair of EDA, Technical University of Munich
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+####################################################################################################
+
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
+include(FetchContent)
+
+FetchContent_Declare(softfloat_srcs
+    GIT_REPOSITORY https://github.com/ucb-bar/berkeley-softfloat-3.git
+    GIT_TAG master
```

Comment:
Maybe we should rather use either a commit hash here or a release tarball from this website: http://www.jhauser.us/arithmetic/SoftFloat.html

Reply:
https://github.com/tum-ei-eda/etiss/pull/132/commits/7c0e4a54c4da544f17d9b18635023271cc66d2aa